Revise Ortac/Dune to minimize yet again boilerplate (#218)
ocaml-gospel/ortac: plugins/dune-rules/README.md

pass to the other plugins and some more information for dune.

Let's say you want use the [Ortac/QCheck-STM] plugin on a module interface
`lib.mli` to generate QCheck-STM tests.

The best way to use Ortac/Dune is in a dune stanza:

```dune
(rule
 (alias runtest)
 (mode promote)
 (action
  (with-stdout-to
   dune.inc
   (run ortac dune qcheck-stm lib.mli))))

(include dune.inc)
```

This stanza assumes that you have written the configuration for
[Ortac/QCheck-STM] in a file named `lib_config.ml` and that the `Lib` module is
part of the `lib` library. It will write the generated tests in `lib_tests.ml`.

> If you want more control, you can use the `--config` `--library` and `--output`

Reviewer: ```suggestion
If you want more control, you can use the `--config`, `--library` and `--output`
```

pass to the other plugins and some more information for dune.

Let's say you want use the [Ortac/QCheck-STM] plugin on a module interface
`lib.mli` to generate QCheck-STM tests.

The best way to use Ortac/Dune is in a dune stanza:

```dune
(rule
 (alias runtest)
 (mode promote)
 (targets dune.inc)
 (action
  (with-stdout-to
   %{targets}
   (run ortac dune qcheck-stm lib.mli))))

(include dune.inc)
```

This stanza assumes that you have written the configuration for
[Ortac/QCheck-STM] in a file named `lib_config.ml` and that the `Lib` module is
part of the `lib` library. It will write the generated tests in `lib_tests.ml`.

> If you want more control, you can use the `--config` `--library` and `--output`
> to give custom name to these files.

Reviewer: ```suggestion
If you want more control, you can use the `--config`, `--library` and `--output`
command-line options to give custom names to these files.
```

pass to the other plugins and some more information for dune.

Let's say you want use the [Ortac/QCheck-STM] plugin on a module interface
`lib.mli` to generate QCheck-STM tests.

The best way to use Ortac/Dune is in a dune stanza:

```dune
(rule
 (alias runtest)

>  (mode promote)
>  (targets dune.inc)
>  (action
>   (with-stdout-to
>    %{targets}
>    (run ortac dune qcheck-stm lib.mli))))

Reviewer: ```suggestion
 (mode promote)
 (action
  (with-stdout-to
   dune.inc
   (run ortac dune qcheck-stm lib.mli))))
```
